R/foreign-keys.R

```diff
@@ -163,15 +162,15 @@ cdm_rm_fk <- function(dm, table, column, ref_table) {
 
 #' Find foreign key candidates in a table
 #'
-#' Which columns are good candidates as a foreign key of a table, referencing
-#' the primary key column of another [`dm`] object's table?
+#' Determine which columns would be good candidates to be used as foreign keys of a table,
```

Comment:
By definition, a foreign key references a p.k. of another table. Can you please rephrase and perhaps split?

Reply:
The original version is already slightly misleading.